server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -173,15 +187,14 @@ export class SyncChefsDataService {
     }
   }
 
-  async syncChefsData(): Promise<void> {
+  async syncChefsData(syncType: SyncTypes): Promise<void> {
```

Comment:
It's better to not update this method and write a separate method (called from sync-chefs-data.ts) using the syncType param!

Reply:
Kindly don't update or change any logic in this method!